### ISSUE TITLE
RegTest can't handle prompts ending in question marks

### ISSUE THREAD

**Issue by spacehobo** [erkyrath/plotex#14]
We use regtest with zvm in dumb-terminal mode, as that is the most reliable way to just grind over the test suite in a makefile on a wide variety of systems.  We're using PunyInform, and it seems the default prompt when you win a game triggers a bit of a timeout at https://github.com/erkyrath/plotex/blob/master/regtest.py#L552-L558

```
> s


  *** You have won ***

In that game you scored 131 out of a possible 131, in 80 turns, earning you the rank of Amateur Larcenist.

Would you like to RESTART, RESTORE, give the FULL score for that game or QUIT? 
```

**Comment by erkyrath** [comment]
That's correct. It's mentioned in the docs somewhere. In cheap mode, it looks for a prompt which is "> " at the beginning of a line. It can't handle any other indication of input.

That's why I added RemGlk mode.

**Comment by curiousdannii** [comment]
If you're using my ZVM, it doesn't have RemGlk mode, but Emglken does.
